agent/acl_test.go

```diff
@@ -258,7 +265,7 @@ var (
 		nodeRWSecret: {
 			token: structs.ACLToken{
 				AccessorID: "efb6b7d5-d343-47c1-b4cb-aa6b94d2f490",
-				SecretID:   nodeROSecret,
+				SecretID:   nodeRWSecret,
```

Comment:
Fixed a previous typo although it doesn't matter in the tests as the secret was already used to lookup the token.